src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/AggregationManager.cs

```diff
@@ -0,0 +1,312 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Runtime.Versioning;
+using System.Security;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.Diagnostics.Metrics
+{
+#if OS_SUPPORT_ATTRTIBUTES
+    [UnsupportedOSPlatform("browser")]
+#endif
+    [SecuritySafeCritical]
+    internal class AggregationManager
+    {
+        public const double MinCollectionTime = 0.1;
```

Comment:
>. [](http://example.com/codeflow?start=49&length=1)

could you please add a comment what the unit of this value?

Reply:
I changed its name to include "Secs"